meta/change_management.md

```diff
@@ -0,0 +1,65 @@
+- Jeremy Swingle
+- Zachary Tinker
+
+During the initial version development of this project, all changes will be made as follows:
+
+1. A change's author should submit the change as a *Pull Request* (PR) to the project's `master` branch
+2. At least 1 developer other than the author should comment on the PR; however, ideally many developers will comment
+3. Discussion, debate, and revisions to the PR happen all within GitHub's PR system
+4. The PR is merged to `master` when any of the following is true:
+    - 1 developer other than the author agrees with the change and there are no disagreements from the other developers after 1 week from PR submission
+    - 2 developers other than the author agree with the change
+
+## Phase 2: Version 1 Quality Assurance
+
+Phase 2 begins once all 4 developers agree that all primary writing and editing as part of phase 1 has concluded. Additional editing may still take place, but at this point, all 4 developers believe they have a working product that, while potentially containing errors, is not likely missing information.
+
+The following are the steps of phase 2:
+
+1. Gryphon conducts an "OCD pass" of all content, making it align exactly with syntax, style, and other requirements
+2. Gryphon submits a PR for review by the other 3 developers
+3. Discussion and debate take place as part phase 1 PRs, and eventually the PR is merged to `master`
+4. Gryphon builds a single *Adobe PDF file* (PDF) for the entire project
+5. Gryphon creates a PR covering the entire project
+
+## Phase 3: CQLT Review and Approval
+
+The PDF will be submitted to the CQLT for review and approval. The 6 quizzing members of the CQLT will have 1 calendar month from the date of submission to review and provide feedback on the submission. Any feedback other than general approval should be made as GitHub comments to the PR covering the entire project.
+
+The 4 developers will respond to any feedback, potentially creating subsequent versions of the output from phase 2 as necessary.
+
+At the end of the 1 month review period of phase 3, if there are no unresolved comments regarding *functional equivalence* (FE) variance, the 6 quizzing members of the CQLT will vote to approve.
+
+## Phase 4: District Ratification
+
+Following the conclusion of phase 3, Zachary will take a resulting final PDF of the whole of the project's content and submit this to all 16 active districts. Gryphon will make available a ratification PR similar to the PR from phase 3.
+
+During phase 4, anyone may provide feedback on the phase 4 PR. The 4 developers will respond to this feedback as they did for phase 3 feedback.
+
+When a *district's coordinator* (DC) confirms to his or her satisfaction that the PDF represents FE to the current rule book, that DC will notify Zachary of his or her district's ratification of the PDF. Once Zachary receives ratification notification from over 66% of the 16 districts (11 or more districts), then phase 4 is complete.
```

Comment:
I still want the "if a district is not confirming their satisfaction of FE, they have to state ALL the reasons why"